bindings/python/tests/test_triplestore.py

```diff
@@ -103,89 +97,113 @@
 
 # Test rdflib triplestore backend
 # -------------------------------
-ts = Triplestore("rdflib")
-assert ts.expand_iri("xsd:integer") == XSD.integer
-assert ts.prefix_iri(RDF.type) == 'rdf:type'
-EX = ts.bind("ex", "http://example.com/onto#")
-assert str(EX) == "http://example.com/onto#"
-ts.add_mapsTo(EX.MyConcept, "http://onto-ns.com/meta/0.1/MyEntity", "myprop")
-ts.add((EX.MyConcept, RDFS.subClassOf, OWL.Thing))
-ts.add((EX.AnotherConcept, RDFS.subClassOf, OWL.Thing))
-ts.add((EX.Sum, RDFS.subClassOf, OWL.Thing))
-assert ts.has(EX.Sum) == True
-assert ts.has(EX.Sum, RDFS.subClassOf, OWL.Thing) == True
-assert ts.has(object=EX.NotInOntology) == False
+if rdflib:
+    ts = Triplestore("rdflib")
+    assert ts.expand_iri("xsd:integer") == XSD.integer
+    assert ts.prefix_iri(RDF.type) == 'rdf:type'
+    EX = ts.bind("ex", "http://example.com/onto#")
+    assert str(EX) == "http://example.com/onto#"
+    ts.add_mapsTo(
+        EX.MyConcept, "http://onto-ns.com/meta/0.1/MyEntity", "myprop")
+    ts.add((EX.MyConcept, RDFS.subClassOf, OWL.Thing))
+    ts.add((EX.AnotherConcept, RDFS.subClassOf, OWL.Thing))
+    ts.add((EX.Sum, RDFS.subClassOf, OWL.Thing))
+    assert ts.has(EX.Sum) == True
+    assert ts.has(EX.Sum, RDFS.subClassOf, OWL.Thing) == True
+    assert ts.has(object=EX.NotInOntology) == False
+
 
+    def sum(a, b):
```

Comment:
"Summarise" seems a bit strange, is this the best term?

Reply:
Probably not. Changed to "Returns the sum of `a` and `b`."